content/app/development/ux/images/_index.nb.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Bilder
+description: Hvordan legge til og endre referanser til bilder.
+toc: true
+weight: 40
+---
+
+## Legge til bilder i applikasjonen
+
+Å legge til bilder gjøres i _FormLayout.json_ ved bruk av bildekomponenten. Alternativ tekst for bildet legges til som en tekst ressurs, som er definert i _resource.[språk].json_.
```

Comment:
Alternativ tekst (for de som ikke ser bilde) er som vanlig en referanse til resource.[språk].json?